### ISSUE TITLE
Hang in `bikeshed update`

### ISSUE THREAD

**Issue by jyasskin** [tabatkins/bikeshed#1746]
With bikeshed-1.6.0:
```
$ bikeshed update
Updating via manifest...
Updating 2 files...
Updated 2/2...

^CTraceback (most recent call last):
  File "…/bin/bikeshed", line 8, in <module>
    sys.exit(main())
  File "…/lib/python3.7/site-packages/bikeshed/cli.py", line 218, in main
    handleUpdate(options, extras)
  File "…/lib/python3.7/site-packages/bikeshed/cli.py", line 246, in handleUpdate
    update.update(anchors=options.anchors, backrefs=options.backrefs, biblio=options.biblio, caniuse=options.caniuse, mdn=options.mdn, linkDefaults=options.linkDefaults, testSuites=options.testSuites, languages=options.languages, wpt=options.wpt, dryRun=constants.dryRun, force=options.force)
  File "…/lib/python3.7/site-packages/bikeshed/update/main.py", line 26, in update
    success = manifest.updateByManifest(path=path, dryRun=dryRun)
  File "…/lib/python3.7/site-packages/bikeshed/update/manifest.py", line 148, in updateByManifest
    asyncio.run(updateFiles(path, newPaths))
  File "…/lib/python3.7/asyncio/runners.py", line 43, in run
    return loop.run_until_complete(main)
  File "…/lib/python3.7/asyncio/base_events.py", line 574, in run_until_complete
    self.run_forever()
  File "…/lib/python3.7/asyncio/base_events.py", line 541, in run_forever
    self._run_once()
  File "…/lib/python3.7/asyncio/base_events.py", line 1750, in _run_once
    event_list = self._selector.select(timeout)
  File "…/lib/python3.7/selectors.py", line 558, in select
    kev_list = self._selector.control(None, max_ev, timeout)
KeyboardInterrupt
```

**Comment by tabatkins** [comment]
...huh.

So I can reproduce the hang on one computer that installed from pip. And I absolutely can't on the dev computer using a local install.

Thanks for the pointer, I'll figure out what's up.

**Comment by tabatkins** [comment]
Ah, good, I can reproduce it with either type of install on my windows machine, and neither type of install on my linux. That suggests I might have a dependency issue? Hm.

**Comment by tabatkins** [comment]
Hmmmmmmm, I notice my linux box has aiohttp `3.6.2`, but windows has `4.0.0a1`. But my requirements.txt clearly says `aiohttp>=3.6.2<4`, which should be precluding `4.0.0a1` from getting installed, wtf.

**Comment by tabatkins** [comment]
And of course I need a *comma* between the version specifiers, ugh. With that fix in place, it successfully chooses 3.6.2, and doesn't hang. Pushing a 1.6.1 release right now, thanks y'all.
